src/validation/managing.officer.corporate.validation.ts

```diff
@@ -45,5 +45,5 @@ export const managingOfficerCorporate = [
   body("contact_email")
     .not().isEmpty().withMessage(ErrorMessages.EMAIL)
     .isLength({ max: 250 }).withMessage(ErrorMessages.MAX_EMAIL_LENGTH)
-    .isEmail().withMessage(ErrorMessages.EMAIL_INVALID_FORMAT)
+    .matches(VALID_EMAIL_FORMAT).withMessage(ErrorMessages.EMAIL_INVALID_FORMAT),
```

Comment:
you could move the "contact_email" validations into email.validation.ts as well, and export it under a different name, contact_email_validations. Similar to what we have done in address.validation.ts, at least the email checks will all be in one file then.

Reply:
Could we even just refactor "contact_email" to be just "email"?

I'm wondering why it's got a different name.

Reply:
Moved contact email, if we want to change the name in the model it would be better to do that separately I don't want to move too far out of scope.